### ISSUE TITLE
http dependencies on file system are not stored in paket-files directory

### ISSUE THREAD

**Issue by jruizaranguren** [fsprojects/Paket#1313]
According to the docs, the following example:

```F#
http http://www.fssnip.net/raw/1M/test1.fs src/test1.fs
```

is stored at the directory `paket-files\www.fssnip.net\src\test.fs`

I'm referencing a local file through:

```F#
http file:///absolute/path/in/windows/to/package.dll package/package.dll
```

But the file is stored at `C:/C_/package/package.dll`

Am I doing something wrong or it is just improvable behavior?

**Comment by forki** [comment]
I guess you are the first person who tried file protocol.

**Comment by mexx** [comment]
Can you try to use `localhost` as host in your url.
```
http file://localhost/absolute/path/in/windows/to/package.dll package/package.dll
```

**Comment by jruizaranguren** [comment]
@mexx, i get the error `The given path's format is not supported.`

**Comment by jruizaranguren** [comment]
Any hint on where to look in code for this? Thanks

**Comment by jruizaranguren** [comment]
Solved in 2.39.6
